<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EyeConfig</class>
 <widget class="QDialog" name="EyeConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Cursor Chaser</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QPushButton" name="applyBut">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>83</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBut">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>83</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitBut">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>83</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="eyesCheck">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Eyes</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="arrowCheck">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Arrow</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="bubbleCheck">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Bubble</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="frameCheck">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Frame</string>
   </property>
  </widget>
  <widget class="QPushButton" name="versionButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Version</string>
   </property>
  </widget>
  <widget class="QPushButton" name="authorButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Author</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBut">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>83</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
